### Question 1.

In [1]:
# 1a) 95% confidence interval
%run mystats
reading_speed = [5, 7, 15, 10, 8, 7, 10, 11, 9, 13, 10, 8, 11, 8, 10, 8, 7, 6, 11, 8]
t_interval(reading_speed, conf=.95)

(7.9749195160742703, 10.225080483925728)

**1b) **
Probability plot
![probability_plot](images/probplot2.png)

** 1c) ** The interval indicates that there is 95% chance that the true population mean lies within the range 7.975 and 10.225. And if we repeatedly sample 20 students and compute this interval, there is a 95% chance that the intervals would contain the mean $\mu$. Stated another way, about 95% of those intervals would contain $\mu$.

In [3]:
# 1d) 97% confidence interval
t_interval(reading_speed, conf=.97)

(7.8391247336019489, 10.36087526639805)

Changing from a 95% confidence interval to a 97% confidence interval merely gives us a wider interval (2.25 vs 2.522), thus also a less precise interval. We can say that there is a 97% chance that the population mean falls within the range 7.839 to 10.361. 

### Question 2.

In [20]:
# Using formula to compute 95% confidence interval
x_bar = 9.10
s = 2.404
n = 20
alpha = .80
p = (alpha + 1) / 2.
# 7.975 10.225 20 .95
# 8.170 10.02  20 .90
t = stats.t(n - 1)
(x_bar - t.ppf(p)*(s/np.sqrt(n)), x_bar + t.ppf(p)*(s/np.sqrt(n)))

(8.3862787163408754, 9.8137212836591239)

In [14]:
alpha = .95
(alpha + 1) / 2.

0.975

### Question 3.

   a) A point estimate for the mean time would be $\bar{y} = 27.6$
   
   b) Standard deviation of point estimate is $s /\sqrt{n} = 3.7 / \sqrt{34} = 0.6345$

In [5]:
# 3c) 95% confidence interval
y_bar = 27.6
s = 3.7
n = 34

t = stats.t(n -1)
(y_bar - t.ppf(.975)*(s/np.sqrt(n)), y_bar + t.ppf(.975)*(s/np.sqrt(n)))

(26.309008968635599, 28.890991031364404)

The office manager should interpret this as saying that there is a 95% chance that the true mean resolution time falls within the range 26.3 to 28.9 minutes. Or that the manager can be 95% confident that cases are being resolved between 26.3 and 28.9 minutes.

### Question 4.

a) True
b) True
c) False
d) False

### Question 5.

In [9]:
# 5a) 99% confidence interval
reading_time = np.array([25, 18, 27, 29, 20, 19, 16, 24, 37, 21, 24, 19, 20, 28, 31, 22])

y_bar = reading_time.mean()
n = len(reading_time)
s = np.sqrt(sum([(y - y_bar) ** 2 for y in reading_time]) / (n - 1))

t = stats.t(n - 1)
(y_bar - t.ppf(.995)*(s/np.sqrt(n)), y_bar + t.ppf(.995)*(s/np.sqrt(n)))

(19.657179514587956, 27.842820485412044)

**5b) ** The interval above is applicable for the population of reading speeds of third graders in the large school district.

**5c) ** Based on the interval from part (a) we can say that we are 95% confident that the average third grader reading speed is between 19.7 and 27.8 minutes.

### Question 6.

In [7]:
# number of renters needed in survey
z = stats.norm().ppf(.96)
E = 97
sigma = (1500 - 45) / 4.

np.ceil(((z ** 2) * (sigma ** 2)) / (E ** 2))

44.0

### Question 7.

In [8]:
# number of renters needed in survey
# for 98% confidence interval

z = stats.norm().ppf(.99)
np.ceil(((z ** 2) * (sigma ** 2)) / (E ** 2))

77.0